### ISSUE TITLE
readxl incorrect where column names are numbers and column type is number

### ISSUE THREAD

**Issue by randyzwitch** [aviks/Taro.jl#40]
Data: http://www.pjm.com/pub/operations/hist-meter-load/1994-hourly-loads.xls

```julia
df1994 = readxl("/Users/randyzwitch/Downloads/1994-hourly-loads.xls", "A1:Z5111")
showcols(df1994)

Exception in thread "main" java.lang.IllegalStateException: Cannot get a text value from a numeric cell
	at org.apache.poi.hssf.usermodel.HSSFCell.typeMismatch(HSSFCell.java:648)
	at org.apache.poi.hssf.usermodel.HSSFCell.getRichStringCellValue(HSSFCell.java:725)
	at org.apache.poi.hssf.usermodel.HSSFCell.getStringCellValue(HSSFCell.java:708)
WARNING: Tried to read column headers, but failed. Set 'headers=false' if you do not have headers
```
In this case, Taro falls back to keeping the first row and putting it into the dataframe, but the result is incorrect. In this case, maybe there is a way to use `!names()` with the first row of the dataframe that does come back, to get the correct result instead of throwing a warning

**Comment by aviks** [comment]
Yes, headers need to be strings right now.

**Comment by randyzwitch** [comment]
Should've made this clearer...I'm working on this too :)

**Comment by aviks** [comment]
I think I have this fixed in #47. Do you have the original files you were using to test? They don't seem to exist on the url above any more.

**Comment by randyzwitch** [comment]
This is the right file name, hopefully I didn't change its structure
[1994-hourly-loads.zip](https://github.com/aviks/Taro.jl/files/2670001/1994-hourly-loads.zip)
